src/player.h

```diff
@@ -59,7 +59,7 @@ namespace Player {
 	/**
 	 * Initializes EasyRPG Player.
 	 */
-	void Init(int argc, char *argv[]);
+	void Init(std::vector<std::string> arguments);
```

Comment:
nit: Here and below: Pass as ``const`` reference ``&``

Reply:
Actually wanted to ask if this would be a good idea to `std::move` the vector.